core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -170,76 +145,45 @@ FunctionExpression getDateFormatExpression() {
   public void adddate() {
     FunctionExpression expr = DSL.adddate(DSL.date(DSL.literal("2020-08-26")), DSL.literal(7));
     assertEquals(DATE, expr.type());
-    assertEquals(new ExprDateValue("2020-09-02"), expr.valueOf(env));
+    assertEquals(new ExprDateValue("2020-09-02"), expr.valueOf());
     assertEquals("adddate(date(\"2020-08-26\"), 7)", expr.toString());
 
     expr = DSL.adddate(DSL.timestamp(DSL.literal("2020-08-26 12:05:00")), DSL.literal(7));
     assertEquals(DATETIME, expr.type());
-    assertEquals(new ExprDatetimeValue("2020-09-02 12:05:00"), expr.valueOf(env));
+    assertEquals(new ExprDatetimeValue("2020-09-02 12:05:00"), expr.valueOf());
     assertEquals("adddate(timestamp(\"2020-08-26 12:05:00\"), 7)", expr.toString());
 
     expr = DSL.adddate(
         DSL.date(DSL.literal("2020-08-26")), DSL.interval(DSL.literal(1), DSL.literal("hour")));
     assertEquals(DATETIME, expr.type());
-    assertEquals(new ExprDatetimeValue("2020-08-26 01:00:00"), expr.valueOf(env));
+    assertEquals(new ExprDatetimeValue("2020-08-26 01:00:00"), expr.valueOf());
     assertEquals("adddate(date(\"2020-08-26\"), interval(1, \"hour\"))", expr.toString());
 
     expr = DSL.adddate(DSL.literal("2020-08-26"), DSL.literal(7));
     assertEquals(DATETIME, expr.type());
-    assertEquals(new ExprDateValue("2020-09-02"), expr.valueOf(env));
+    assertEquals(new ExprDateValue("2020-09-02"), expr.valueOf());
     assertEquals("adddate(\"2020-08-26\", 7)", expr.toString());
 
     expr = DSL.adddate(DSL.literal("2020-08-26 12:05:00"), DSL.literal(7));
     assertEquals(DATETIME, expr.type());
-    assertEquals(new ExprDatetimeValue("2020-09-02 12:05:00"), expr.valueOf(env));
+    assertEquals(new ExprDatetimeValue("2020-09-02 12:05:00"), expr.valueOf());
     assertEquals("adddate(\"2020-08-26 12:05:00\", 7)", expr.toString());
 
     expr = DSL
         .adddate(DSL.literal("2020-08-26"), DSL.interval(DSL.literal(1), DSL.literal("hour")));
     assertEquals(DATETIME, expr.type());
-    assertEquals(new ExprDatetimeValue("2020-08-26 01:00:00"), expr.valueOf(env));
+    assertEquals(new ExprDatetimeValue("2020-08-26 01:00:00"), expr.valueOf());
     assertEquals("adddate(\"2020-08-26\", interval(1, \"hour\"))", expr.toString());
 
     expr = DSL
         .adddate(DSL.literal("2020-08-26"), DSL.interval(DSL.literal(1), DSL.literal("day")));
     assertEquals(DATETIME, expr.type());
-    assertEquals(new ExprDateValue("2020-08-27"), expr.valueOf(env));
+    assertEquals(new ExprDateValue("2020-08-27"), expr.valueOf());
     assertEquals("adddate(\"2020-08-26\", interval(1, \"day\"))", expr.toString());
-
-    when(nullRef.type()).thenReturn(DATE);
```

Comment:
I'm curious on what these removed asserts actually do. I noticed them as I've been working on DateTime stuff, but I don't actually know what they are supposed to be testing.

Reply:
They handle `nullMissingHandling`, which is tested in `FunctionDSLnullMissingHandlingTest`. Thanks Max by adding those tests.

This specific line mocks type of `nullRef` to let repository pick a function signature.
The null/missing validation actually doesn't test the function and it is not needed to repeat it 100500 times for each function.